### PR TITLE
Doc/ Enhance English Style Guide

### PR DESCRIPTION
**Pull request by wudifeixue**
Oh my Kagami, thank you for these wonderful guides and documents!
I definitely appreciate your effort for such great work!

I happen to see there are a few errors in the English one and correct them as I look through. This pull request is in an attempt to make the document even better.
* Code should always be code in the sense of programming. [source](http://english.stackexchange.com/questions/20455/is-it-wrong-to-use-the-word-codes-in-a-programming-context)
* I change all esLint or eslint to ESLint since it brand itself as so? I excluded SublimeLinter-eslint due to their branding of the plugin is lowercase. 
* There are some other minor mistake corrections in this pull request such as spelling and grammar.

### CODE REVIEW

File: docs/style-guide.md [hunk 1]
```diff
@@ -1,30 +1,30 @@
...
 
 ``` javascript
 const {api_maparea_id, api_mapinfo_no} = postBody
 const mapId = `${api_maparea_id}${api_mapinfo_no}`
 ```
-Function naming rules are the same as variables.
+Function use the same naming rules as variables.
 
-for constants, use SNAKE_CASE: `MAX_RETRY`，`APPDATA_PATH`.
+For constants, use UPPERCASE_SNAKE_CASE: `MAX_RETRY`，`APPDATA_PATH`.
```
Comment (KagamiChan) [inline docs/style-guide.md]: SANKE_CASE has clearly explained itself

File: docs/style-guide.md [hunk 2]
```diff
@@ -1,30 +1,30 @@
 # Code style guide
 
-The document introduces the style guide for codes submited to poi. Codes of good and coherent style is friendly to other contributors and yourself in the future. Please follow them.
+This document introduces the style guide for code submitted to poi. Good and coherent code style is friendly to other contributors and yourself in the future. Please follow them.
```
Comment (KagamiChan) [inline docs/style-guide.md]: should it be the codes that are friendly to people?

File: docs/style-guide.md [hunk 3]
```diff
@@ -1,30 +1,30 @@
...
 const mapId = `${api_maparea_id}${api_mapinfo_no}`
 ```
-Function naming rules are the same as variables.
+Function use the same naming rules as variables.
 
-for constants, use SNAKE_CASE: `MAX_RETRY`，`APPDATA_PATH`.
+For constants, use UPPERCASE_SNAKE_CASE: `MAX_RETRY`，`APPDATA_PATH`.
 
-For class or react element, use CamelCase: `FileWriter`，`ShipView`.
+For class or react element, use PrimalCase: `FileWriter`，`ShipView`.
```
Comment (KagamiChan) [inline docs/style-guide.md]: Seems PrimalCase is a far less used word

File: docs/style-guide.md [hunk 4]
```diff
@@ -44,8 +44,8 @@ const fs = Promise.promisifyAll(require('fs-extra'))
 
 ### Leave a blank line (EOL) at the end of file
 
-### Comments
-Feel free to leave comments where necessary.
+### Code Comments
+Feel free to comment your code when necessary.
```
Comment (KagamiChan) [inline docs/style-guide.md]: `where necessary` means space/location in codes, it is not a mistake

File: docs/style-guide.md (hunk 3 above)
Comment (wudifeixue) [inline docs/style-guide.md]: You are right, I knew I could have a better name for it, and it should be Pascal case.

File: docs/style-guide.md (hunk 2 above)
Comment (wudifeixue) [inline docs/style-guide.md]: To be honest, even after the changes I still feel awkward after I read this part out. This definitely has a lot of room for improvement.

File: docs/style-guide.md (hunk 1 above)
Comment (wudifeixue) [inline docs/style-guide.md]: Yeah, I agree. 😄

File: docs/style-guide.md (hunk 4 above)
Comment (wudifeixue) [inline docs/style-guide.md]: Yes I know, it is a bit confusing when I read it first, so I changed the wording in an attempt to enhance the clarity. We can definitely make it better.

File: docs/style-guide.md (hunk 2 above)
Comment (wudifeixue) [inline docs/style-guide.md]: Let's try this.   
This document is the style guide for poi. Good and coherent style is friendly to other contributors and yourself in the future. Please follow them.

File: docs/style-guide.md (hunk 1 above)
Comment (wudifeixue) [inline docs/style-guide.md]: How do I change it to 
`For constants, use SNAKE_CASE:`
at this moment?

File: docs/style-guide.md (hunk 3 above)
Comment (KagamiChan) [inline docs/style-guide.md]: CamelCase and camelCase are OK, see https://en.wikipedia.org/wiki/Camel_case

File: docs/style-guide.md (hunk 1 above)
Comment (KagamiChan) [inline docs/style-guide.md]: use git commit --amend and force update your branch, or squash your commits (force update is inevitable though)

File: docs/style-guide.md (hunk 4 above)
Comment (KagamiChan) [inline docs/style-guide.md]: commenting code sometimes means changing codes into comments, it is a little ambiguous

File: docs/style-guide.md (hunk 2 above)
Comment (KagamiChan) [inline docs/style-guide.md]: well, we can use another word than friendly, `will be of great help` so something

File: docs/style-guide.md (hunk 4 above)
Comment (wudifeixue) [inline docs/style-guide.md]: That's right, so we rephrase this to:

`Please feel free to write comments in your code where necessary.`

?

**Comment by wudifeixue** [comment]
Applied changes based on the review with @KagamiChan .
